src/main/java/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package service.impl;
+
+import service.FruitShopService;
+import service.WriteToFile;
+import service.WriteToStorage;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final WriteToStorage writeToStorage = new WriteToStorageImpl();
+    private final WriteToFile writeFromStorage = new WriteToFileImpl();
+
+    @Override
+    public void getResult(String fromFileName, String toFileName) {
+        writeToStorage.writeToDataBase(fromFileName);
+        writeFromStorage.writeFromDataBase(toFileName);
+    }
+}
```

Comment:
this service should contain all the other services, not just reading and writing. This signals you have violations of SRP. Parsing data is separate from reading it, applying transactions is separate from parsing, writing of results is separate from applying transactions. Our code should be modular, so the changes to one module don't affect others. Please review image of proposed architecture in Readme of this task